cfme/fixtures/vm.py

```diff
@@ -49,7 +49,11 @@ def _create_vm(request, template_type, provider, vm_name):
     @request.addfinalizer
     def _cleanup():
         vm_obj.cleanup_on_provider()
-        provider.refresh_provider_relationships()
+        try:
+            provider.refresh_provider_relationships()
+        except Exception as e:
```

Comment:
Could you please be more specific about which Exception can it be?

Reply:
Sure. And then also catch the more specific LookupError instead of just an Exception in _cleanup().

Reply:
I don't know why we'd want the else clause, however.

Reply:
Changing exception but not adding else.